### PR TITLE
fix the typo for cloud_nonuser_sql

### PR DESCRIPTION
**Pull request by gavinyue**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

File: pkg/util/trace/impl/motrace/report_statement.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func StatementInfoFilter(i Item) bool {
 
 	// Check SqlSourceType
 	switch statementInfo.SqlSourceType {
-	case "internal_sql", "external_sql", "non_cloud_user":
+	case "internal_sql", "external_sql", "cloud_nonuser_sql":
```
Comment (xzxiong) [inline pkg/util/trace/impl/motrace/report_statement.go:122]: 能否和 彭振 或王剑 讨论一下，将 `internalSql` 等常量 放到 `pkg/frontend/const` 或 `pkg/frontend/type`  的目录下？
我们一直用字符串也不是个事

Comment (xzxiong) [reply pkg/util/trace/impl/motrace/report_statement.go:122]: 现在是在这
https://github.com/matrixorigin/matrixone/blob/8870ea87a96828862413e88f380c5bf5159fa5ad/pkg/frontend/types.go#L105-L117

File: pkg/frontend/types.go [hunk 2]
```diff
@@ -108,10 +108,7 @@ Disguise the COMMAND CMD_FIELD_LIST as sql query.
 const (
 	cmdFieldListSql    = "__++__internal_cmd_field_list"
 	cmdFieldListSqlLen = len(cmdFieldListSql)
-	internalSql        = "internal_sql"
 	cloudUserSql       = "cloud_user_sql"
```
Comment (xzxiong) [inline pkg/frontend/types.go:111]: 建议这个也搬过去

**Review by xzxiong — APPROVED** [review]
LGTM

**Review by heni02 — APPROVED** [review]

**Comment by guguducken** [comment]
@mergifyio refresh

**Pull request merged**